workspace/apps/pidp/src/app/features/shell/pages/support-error/support-error.page.html

```diff
@@ -0,0 +1,18 @@
+<ui-dashboard-header
+  [theme]="headerConfig.theme"
+  [allowMobileToggle]="headerConfig.allowMobileToggle"></ui-dashboard-header>
+<div>
+  <h1>An error has occurred</h1>
+  <p>
+    Need support? Contact
+    <ui-anchor
+      scheme="mailto"
+      href="provideridentityportal@gov.bc.ca"></ui-anchor>
```

Comment:
You'll want to provide this through a public member variable that is populated from environment